js/helpers/OTHelper.js

```diff
@@ -738,9 +738,34 @@
       return getStream('Audio', elem);
     }
 
+    /**
+     * Chrome has problems getting the audio from some videos with HTMLMediaElement.captureStream()
```

Comment:
Can you be a little more specific about chrome having problems?
Do you have a ticket?

It might be a good idea to add it here to track it so we can, eventually, remove this workaround

Reply:
@albertoruibal could you please add 'OPENTOK-37378' as a reference in the comment?
@robjperez since it looks like an opentok specific bug, I've added an internal ticket that we can reference.